src/ophys_etl/transforms/event_detection.py

```diff
@@ -0,0 +1,571 @@
+import argschema
+import h5py
+import os
+import numpy as np
+import json
+import multiprocessing
+import warnings
+import marshmallow as mm
+from typing import Tuple
+from scipy.ndimage.filters import median_filter
+from scipy.signal import resample_poly
+from scipy.stats import median_abs_deviation
+from functools import partial
+from FastLZeroSpikeInference import fast
+from ophys_etl.resources import event_decay_lookup_dict as decay_lookup
+from ophys_etl.schemas.fields import H5InputFile
+
+
+class EventDetectionException(Exception):
+    pass
+
+
+class EventDetectionInputSchema(argschema.ArgSchema):
+    log_level = argschema.fields.Str(required=False, default="INFO")
+    movie_frame_rate_hz = argschema.fields.Float(
+        required=True,
+        description=("frame rate of the ophys video / trace. "
+                     "used to upsample slower rates to 31Hz."))
+    full_genotype = argschema.fields.Str(
+        required=False,
+        description=("full genotype of the specimen. Used to look "
+                     "up characteristic decay time from "
+                     "ophys_etl.resources.event_decay_time_lookup"))
+    decay_time = argschema.fields.Float(
+        required=False,
+        description=("characteristic decay time [seconds]"))
```

Comment:
Technically, single line string doesn't require parentheses

Reply:
force-of-habit. These comments tend to be multi-line.
fixed.